specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -5061,6 +5616,27 @@
         }
       }
     },
+    "ReplicationDetailsList": {
+      "description": "The list of replication details.",
+      "type": "object",
+      "properties": {

```

Comment:
Please mark properties `readOnly` wherever applicable.

Reply:
Need some help understanding this comment. I left it out of the current iteration. Once I get more clarification, I will add this.

Reply:
If the model properties must have public getters, `readOnly` flag is necessary. If not, it must be set for each of the properties of the model. This ensures better quality SDK

Reply:
Thanks for explaining. All the model properties for our service are public getters.